CMakeLists.txt

```diff
@@ -867,8 +867,8 @@ endfunction()
 # need to be defined when cross-compiling.
 if(NOT CMAKE_SYSTEM_NAME STREQUAL "Darwin")
   if(SWIFT_BUILD_STDLIB OR SWIFT_BUILD_SDK_OVERLAY)
-    swift_icu_variables_set("${SWIFT_HOST_VARIANT_SDK_default}"
-                            "${SWIFT_HOST_VARIANT_ARCH_default}"
+    swift_icu_variables_set("${SWIFT_PRIMARY_VARIANT_SDK}"
+                            "${SWIFT_PRIMARY_VARIANT_ARCH}"
```

Comment:
Worth updating the comment too?

Reply:
I'm happy to do so, though the current comment seems correct?  What did I overlook or what would make it clearer?

Reply:
Maybe I'm misunderstanding this change, but "assuming we are compiling for the build as the host" seems incorrect now.